app/helpers/external_users/claim_types_helper.rb

```diff
@@ -11,5 +11,15 @@ def claim_type_prompt_heading_for(user)
         t('external_users.claim_heading')
       end
     end
+
+    def claim_type_page_header(user)
+      if user.persona.has_roles?('advocate')
+        t('.page_title_advocate')
+      elsif user.persona.has_roles?('litigator')
+        t('.page_title_litigator')
+      else
+        t('.page_title_generic')
+      end
```

Comment:
reminder: This block seems odd, so I'm going to look at the logic

Reply:
It's as I thought.  If a provider has both advocate *and* litigator roles, the heading will _always_ return advocate

A better match may be:
```ruby
      if user.persona.roles.eql?['advocate']
        t('.page_title_advocate')
      elsif user.persona.roles.eql?['litigator']
        t('.page_title_litigator')
      else
        t('.page_title_generic')
      end

Reply:
@cleargif this will still fail,  it needs to be ```persona.roles.eql?['advocate']``` 
`has_roles?` is the problem, not the brackets